### PR TITLE
Sunfury Buildings & Icons

### PR DESCRIPTION
**Pull request by HerrDave**
- Updated all Sunfury buildings except Well of Eternity
- Removed now unused Sunfury assets
- Updated Sunfury production building names to fit new models
- Fixed Windrunner's Spire icon

*Note
For some reason I got a pull pop-up saying I cannot automatically pull, but can still do so manually?
First time I've seen that, not sure why.

### CODE REVIEW

File: mapdata/WarcraftLegacies/Units/-17920_-23552.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/Units/-17920_-23552.json:1]: @HerrDave Looks like this PR adds a bunch of new unit placements on the map. Is that intentional and if so, what are they?

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Just one question to be answered, otherwise looks good.

**Comment by YakaryBovine** [comment]
Marked this as a Draft because:
* Sebas needs to look at it before I can do a technical review
* The testing units in the bottom left corner need to be removed

**Comment by SebastianTanoni** [comment]
Units removed, made the small changes i want, everything looks good on my end

File: mapdata/WarcraftLegacies/Units/-17920_-23552.json (hunk 1 above)
Comment (SebastianTanoni) [reply mapdata/WarcraftLegacies/Units/-17920_-23552.json]: i think they are the testing units, i removed them

**Comment by YakaryBovine** [comment]
I'll just need to take another look at this post-changes.

**Review by YakaryBovine — APPROVED** [review]
Looking good!

**Pull request merged**